packages/core/src/lib/button/button.svelte

```diff
@@ -64,7 +64,7 @@ $: handleDisabled = preventHandler(disabled);
         variant === 'primary' && !disabled,
       'border-gray-9 bg-gray-9 text-white hover:border-black hover:bg-black active:bg-[#000]':
         variant === 'dark' && !disabled,
-      'active-border-[rgba(0,0,0,0.08)] border-transparent text-default hover:bg-[rgba(0,0,0,0.04)] active:bg-[rgba(0,0,0,0.08)]':
+      'border-transparent text-default hover:bg-ghost-light active:border-ghost-medium active:bg-ghost-medium':
```

Comment:
Old bug in prime that was carried into core: active state has not been applied on ghost buttons because of an `active-` vs `active:` typo